internal/pkg/cli/storage_init.go

```diff
@@ -692,65 +691,63 @@ func (o *initStorageOpts) readWorkloadType() error {
 }
 
 type addonBlob struct {
-	name        string
 	description string
 	blob        encoding.BinaryMarshaler
+	path        string
 }
 
 func (o *initStorageOpts) addonBlobs() ([]addonBlob, error) {
-	templateBlob, err := o.newAddonTemplate()
+	switch {
+	case o.lifecycle == lifecycleWorkloadLevel && o.storageType == s3StorageType:
+		return o.wkldS3AddonBlobs()
+	case o.lifecycle == lifecycleWorkloadLevel && o.storageType == dynamoDBStorageType:
+		return o.wkldDDBAddonBlobs()
+	case o.lifecycle == lifecycleWorkloadLevel && o.storageType == rdsStorageType:
+		return o.wkldRDSAddonBlobs()
+	case o.lifecycle == lifecycleEnvironmentLevel && o.storageType == s3StorageType:
+		return o.envS3AddonBlobs()
+	case o.lifecycle == lifecycleEnvironmentLevel && o.storageType == dynamoDBStorageType:
+		return o.envDDBAddonBlobs()
+	case o.lifecycle == lifecycleEnvironmentLevel && o.storageType == rdsStorageType:
+		return o.envRDSAddonBlobs()
+	}
+	return nil, fmt.Errorf("storage type %s doesn't have a CF template", o.storageType)
```

Comment:
```suggestion
	return nil, fmt.Errorf("storage type %s doesn't have a CloudFormation template", o.storageType)
```
To avoid acronyms that users might not be familiar with